code-of-conduct.qmd

```diff
@@ -35,13 +35,14 @@ If the slides are shared or hosted by NHS-R Community, these should also contain
 
 The NHS-R Community has a core development team which consists of:
 
-* [Zoë Turner](mailto:zoe.turner3@nhs.net) - who is also the lead investigator of the NHS-R Community Way and is responsible for enforcing the Code of Conduct
+* [Claire Welsh](claire.welsh8@nhs.net) - NHS-R Community 
+<span style="color:grey;">* [Zoë Turner](mailto:zoe.turner3@nhs.net) - who is also the lead investigator of the NHS-R Community Way and is responsible for enforcing the Code of Conduct (On secondment) </span>
 
 Escalation, as detailed in this book, are to [Chris Beeley](chris.beeley1@nhs.net), a senior manager in The Strategy Unit. 
 
-The [central email](#contact-us) is accessed by Zoë Turner and administrative/support staff in The Strategy Unit.
+The [central email](#contact-us) is accessed by Claire Welsh, Zoë Turner and administrative/support staff in The Strategy Unit.
 
-Zoë Turner is an employee of NHS Midlands and Lancashire Commissioning Support Unit and is not excluded from this Code of Conduct but, if required, contact can be made with the Senior Management team within The Strategy Unit in the same organisation through [strategy.unit@nhs.net](mailto:strategy.unit@nhs.net) which neither have access to.
+Claire Welsh and Zoë Turner are employees of NHS Midlands and Lancashire Commissioning Support Unit and are not excluded from this Code of Conduct but, if required, contact can be made with the Senior Management team within The Strategy Unit in the same organisation through [strategy.unit@nhs.net](mailto:strategy.unit@nhs.net) which neither have access to.
```

Comment:
To save changing in the future if the project members is defined in a broad sense of all Strategy Unit employees then this could just say project members?